### DISCUSSION TITLE
STM32G030K8T6TR PC14 & PC15 gpio

### DISCUSSION THREAD

**Discussion by barbiani** [stm32duino/Arduino_Core_STM32#2248] [Q&A] [answered]
Hello everybody,

I have a board with signals going to PC14 & 15 on this mcu and I'd like to make PC14 an input and PC15 an output.

It seems that the RTC must be configured to make the gpio available to the arduino calls, but I am not finding the correct way.

Thank you.

**Comment by fpistm** [comment, marked as answer]
Hi @barbiani 

to setup the pin in input or output, simply use pinMode api:
https://www.arduino.cc/reference/en/language/functions/digital-io/pinmode/

No need of RTC.
